RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -4,122 +4,92 @@
-                                                                     ])
-                  )
-
+        func=lambda outputCommands: outputCommands.extend(['keep *_zdcreco_*_*',
+                                                           'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                                           'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                                           'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*'])
+        )
 from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
-egamma_lowPt_exclusive.toModify( RecoLocalCaloAOD,
+egamma_lowPt_exclusive.toModify( RecoLocalCaloAOD, 
     outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_towerMaker_*_*',
-						        'keep *_zdcreco_*_*',
+                                                        'keep *_zdcreco_*_*',
                                                         'keep ZDCDataFramesSorted_hcalDigis_*_*',
                                                         'keep ZDCDataFramesSorted_castorDigis_*_*',
                                                         'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*'])
 
+#RECO content
+RecoLocalCaloRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring('keep *_hbhereco_*_*',
+                                           'keep *_hbheprereco_*_*',
+                                           'keep *_hfprereco_*_*',
+                                           'keep *_hfreco_*_*',
+                                           'keep *_horeco_*_*',
+                                           'keep HBHERecHitsSorted_hbherecoMB_*_*',
+                                           'keep HORecHitsSorted_horecoMB_*_*',
+                                           'keep HFRecHitsSorted_hfrecoMB_*_*',
+                                           'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                           'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                           'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*',
+                                           'keep ZDCRecHitsSorted_zdcreco_*_*')
+)
+RecoLocalCaloRECO.outputCommands.extend(RecoLocalCaloAOD.outputCommands)
+phase2_hgcal.toModify( RecoLocalCaloRECO, 
+    outputCommands = RecoLocalCaloRECO.outputCommands + ['keep *_HGCalRecHit_*_*',
+                                                         'keep recoCaloClusters_hgcalLayerClusters_*_*',
+                                                         'keep *_hgcalLayerClusters_timeLayerCluster_*',
+                                                         'keep *_hgcalLayerClusters_InitialLayerClustersMask_*',
+                                                         'keep *_hgcalMultiClusters_*_*',
+                                                         'keep *_iterHGCalMultiClusters_*_*'])
+phase2_hfnose.toModify( RecoLocalCaloRECO, 
```

Comment:
Nothing to add here wrt RecoLocalCaloAOD

Maybe you can just limit this modification at a placeholder:
```PYTHON
phase2_hfnose.toModify( RecoLocalCaloRECO, 
    outputCommands = RecoLocalCaloRECO.outputCommands)
```
(that could also be left commented out)

Reply:
But I think that even better and cleaner could be to simply remove it, as it was done for HI and LowPt customizations